TypeCobol.Analysis/Cfg/ControlFlowGraphBuilder.cs

```diff
@@ -1327,12 +1332,68 @@ private void HandlePerformsWithTestAfter()
             }
         }
 
+        /// <summary>
+        /// Traverse the CFG to detect unreachable code blocks.
+        /// </summary>
+        private void DetectUnreachableCode()
+        {
+            //DFS to flag all reachable statements, the hashset is needed because we also traverse BasicBlockForNodeGroups
+            HashSet<int> visitedBlocks = new HashSet<int>();
+            this.CurrentProgramCfgBuilder.Cfg.DFS(FlagReachableStatements);
+
+            //Collect unreachable blocks from remaining unreachable statements
+            HashSet<int> unreachableBlocks = new HashSet<int>();
```

Comment:
We can also store the unreachableBlocks information as a result in the related CFG? beacuse here this informatiion is lost, if one does not check each diagnostic.

Reply:
Ok. Done here 1cc51fbf07085027fd5a1f80c40f7099af8c49c6.